libmuscle/python/libmuscle/manager/logger.py

```diff
@@ -125,3 +125,36 @@ def log_message(
                 extra={
                     'instance': instance_id,
                     'iasctime': timestamp.to_asctime()})
+
+
+def last_lines(file: Path, count: int) -> str:
+    """Utility function that returns the last lines of a text file.
+
+    This opens the file and returns the final `count` lines. It reads
+    at most 10000 bytes, to avoid memory problems if the file contains
+    e.g. a large amount of binary data.
+
+    Args:
+        file: The file to read
+        count: Number of lines to read
+
+    Return:
+        A string of at most 10000 bytes, containing at most `count`
```

Comment:
```suggestion
        A string of at most 5,000 bytes, containing at most `count`
```

Reply:
Doh, I wanted to extend it to 10k, and updated the comment, but not the implementation :smile:. I'll change it, but I'll change the other number...